src/styles/blocks/price.scss

```diff
@@ -0,0 +1,19 @@
+.price {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 18px;
+  line-height: 18px;
+
+  &__title {
+    font-size: 12px;
+    color: $secondary-color;
```

Comment:
The variable `$secondary-color` is used for the `color` property of the `&__title` class. Ensure that this variable is defined elsewhere in your SCSS files. If it's not defined, you need to declare it with a valid color value before using it.